packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -349,10 +349,10 @@ export interface ISummarizeHeuristicData {
     readonly lastSuccessfulSummary: Readonly<ISummarizeAttempt>;
 
     /**
-     * Initializes lastAttempt and lastSuccessfulAttempt based on the last summary.
+     * Updates lastAttempt and lastSuccessfulAttempt based on the last summary.
      * @param lastSummary - last ack summary
      */
-    initialize(lastSummary: ISummarizeAttempt): void;
+    updateWithPendingSummaryAckInfo(lastSummary: Partial<ISummarizeAttempt>): void;
```

Comment:
nit: updateWithLastSummaryInfo?

Reply:
Also, why make the param Partial? We are passing in all the required properties right.

Reply:
Good catch - I was going to make it partial and we do not need it anymore.